lib/SPIRV/Mangler/FunctionDescriptor.h

```diff
@@ -10,8 +10,8 @@
  * Contributed by: Intel Corporation.
  */
 
-#ifndef __FUNCTION_DESCRIPTOR_H__
-#define __FUNCTION_DESCRIPTOR_H__
+#ifndef LLVM_TOOLS_SPIRV_LLVM_TRANSLATOR_LIB_SPIRV_MANGLER_FUNCTIONDESCRIPTOR_H
```

Comment:
I'm not sure about this change.
LLVM_TOOLS_SPIRV_LLVM_TRANSLATOR_LIB_SPIRV_MANGLER_FUNCTIONDESCRIPTOR_H doesn't seem to be a good name for the guard. Mangler library is a separate library not related to SPIR-V translator, so I don't think we should use notion of SPIR-V translator in this library. Original sources of Mangler library are hosted here: https://github.com/KhronosGroup/SPIR-Tools/tree/master/spir-tools/spir_name_mangler.
Will it help if we move this library out of SPIR-V translator sources?

Reply:
Ah, I didn’t know about that. I’m not sure how we should handle that; we can always manually revert those changes and ensure that the Mangler will be ignored by clang-format and clang-tidy.

Reply:
I see that all headers have got such long header guards. All of them start with `LLVM_TOOLS_SPIRV_LLVM_TRANSLATOR_LIB_SPIRV_` prefix.
I think we should shorten it to `SPIRV_` and disable `llvm-header-guard` check/fix in .clang-tidy

Reply:
I’ll do that. And it will be easy to change back to that long prefix if asked to do so when moving in the LLVM umbrella.